rosetta/docs/GPU_performance.md

```diff
@@ -107,6 +107,7 @@ The below enables CUDA Graph suppport for JAX/XLA workloads, and is enabled by d
 
 The following flag removes extra copies introduced by DUS (dynamic update slice) when used in conjunction with custom NVIDIA kernels (like cuBLAS for GEMMs)
```

Comment:
```suggestion
The following flag removes extra copies introduced by DUS (dynamic update slice) when used in conjunction with custom NVIDIA kernels (like cuBLAS for GEMMs). This happens in particular when used with scan operations.
```

Reply:
I will add a warning to users when using multiple experimental different flags